htmltools/_core.py

```diff
@@ -275,26 +274,26 @@ class TagList(List[TagNode]):
     def __init__(self, *args: TagChild) -> None:
```

Comment:
From https://docs.python.org/3/library/collections.html#collections.UserList:

> Subclasses of [UserList](https://docs.python.org/3/library/collections.html#collections.UserList) are expected to offer a constructor which can be called with either no arguments or one argument. List operations which return a new sequence attempt to create an instance of the actual implementation class. To do so, it assumes that the constructor can be called with a single parameter, which is a sequence object used as a data source.

Reply:
Please unit test `TagList.concat()` and make sure the returned object is also a `TagList`.

Reply:
Make sure radd and add both return tag lists

Reply:
* Added test for constructor with no args, single arg, multiple args
* Added support/tests for `add` and `radd`
* Added basic tests for all list methods described in [docs.python.org/3/library/stdtypes.html#typesseq-common](https://docs.python.org/3/library/stdtypes.html#typesseq-common)

✅